notebooks/tfx_pipelines/guided_projects/tfx_guided_project_vertex.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Convert to a text/markdown block, shows up as a code block. Could also update the instruction to say that json is found in PROJECT_DIR, since current working directory could be confusing and refer to where the notebook is currently running.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Done!</p>